scene/gui/text_edit.cpp

```diff
@@ -4913,24 +4913,28 @@ String TextEdit::get_selection_text() const {
 
 String TextEdit::get_word_under_cursor() const {
 
-	int prev_cc = cursor.column;
-	while (prev_cc > 0) {
-		bool is_char = _is_text_char(text[cursor.line][prev_cc - 1]);
-		if (!is_char)
-			break;
-		--prev_cc;
-	}
+	if (cursor.column >= 0) {
```

Comment:
We could save the extra indent and git diff with:
```cpp
if (cursor.column < 0) {
	return "";
}
```

I wonder why we would ever want `cursor.column` to take negative values though, but that's beyond the scope of this PR, I see many checks for `cusor.column >= 0` or `> 0` in existing code.

Reply:
It being negative is another case of #26192, I have a feeling its being set as such [here](https://github.com/godotengine/godot/blob/master/scene/gui/text_edit.cpp#L4090).